docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1101,67 +1101,113 @@ spec:
-        <name>Time</name>
-        <value>1619008605</value>
-      </metric>
-      <metric type="string" context="host">
-        <name>VirtualizationVendor</name>
-        <value>kubevirt.io</value>
-      </metric>
-    </metrics>
+In order to expose the `downwardMetrics` to the VMs, there are two methods supported: `disk` and `virtio-serial port`.
 
 > **Note:** The **DownwardMetrics** feature gate
 > [must be enabled](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
-> to use this volume. Available starting with KubeVirt v0.42.0.
+> to use the metrics. Available starting with KubeVirt v0.42.0.
+ 
+#### Disk
+A volume is created, and it is exposed to the guest as a raw block volume.
+
+Example:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: metrics
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora:latest
+  - name: metrics
+    downwardMetrics: {}
+```
+
+#### Virtio-serial port
+It creates a virtio-serial port named `org.github.vhostmd.1` inside the VM. The metrics may be requested and fetched 
+through this port.
+
+To enable the `downwardMetrics` within a virtio-serial port, the spec `spec.domain.devices.downwardMetrics: {}` has to be
+added to the VM definition.
+
+Example:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      downwardMetrics: {}
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora:latest
+```
+
+The `vm-dump-metrics` tool can be used to read the `downwardMetrics` shared with both methods described:
+```xml
+$ sudo dnf install -y vm-dump-metrics
+$ sudo vm-dump-metrics
+<metrics>
+  <metric type="string" context="host">
+    <name>HostName</name>
+    <value>node01</value>
+[...]
+  <metric type="int64" context="host" unit="s">
+    <name>Time</name>
+    <value>1619008605</value>
+  </metric>
+  <metric type="string" context="host">
+    <name>VirtualizationVendor</name>
+    <value>kubevirt.io</value>
+  </metric>
+</metrics>
+```
+
+> **Note:** The tool `vm-dump-metrics` provides the option `--virtio` in case the virtio-serial port is used. 
+> Within the full command `vm-dump-metrics --virtio`, the `DownwardMetrics` can be fetched directly from virtio-serial 
```

Comment:
This phrase is not clear. You can also refer to  `vm-dump-metrics --help` for help.

Reply:
I've dropped some details of the sentence and included the reference to the help info.